doc/how_to/get_started.rst

```diff
@@ -337,109 +368,221 @@ can print the units found using:
     Units found by spyking-circus2: [0 1 2 3 4 5 6 7 8 9]
 
 
-If a sorter is not installed locally, we can also avoid to install it
-and run it anyways, using a container (Docker or Singularity). For
-example, let’s run ``Kilosort2`` using Docker:
+If a sorter is not installed locally, we can also avoid installing it
+and run it anyways, using a container (Docker or Singularity). To do
+this, you will need to install Docker. More information
+`here <https://spikeinterface.readthedocs.io/en/latest/modules/sorters.html?highlight=docker#running-sorters-in-docker-singularity-containers>`__.
+Let’s run ``Kilosort2`` using Docker:
 
-.. code:: ipython
+.. code:: ipython3
 
-    sorting_KS2 = ss.run_sorter(sorter_name="kilosort2", recording=recording_preprocessed,
-                                docker_image=True, verbose=True)
+    sorting_KS2 = ss.run_sorter(sorter_name="kilosort2", recording=recording_preprocessed, docker_image=True, verbose=True)
     print(sorting_KS2)
 
 
 .. parsed-literal::
 
+    installation_mode='auto' switching to installation_mode: 'dev'
     Starting container
-    Installing spikeinterface from sources in spikeinterface/kilosort2-compiled-base
-    Installing dev spikeinterface from local machine
-    Installing extra requirements: ['neo', 'mearec']
+    Installing spikeinterface with folder in container
+    Installing neo with pypi in container
+    Installing mearec with pypi in container
     Running kilosort2 sorter inside spikeinterface/kilosort2-compiled-base
     Stopping container
+    KiloSortSortingExtractor: 19 units - 1 segments - 32.0kHz
 
-.. parsed-literal::
 
-    KiloSortSortingExtractor: 19 units - 1 segments - 32.0kHz
+For postprocessing SpikeInterface pairs recording and sorting objects
+into a ``SortingAnalyzer`` object. The ``SortingAnalyzer`` can be loaded
+in memory or saved in a folder. Here, we save it in binary format.
 
+.. code:: ipython3
+
+    analyzer_TDC = si.create_sorting_analyzer(sorting=sorting_TDC, recording=recording_preprocessed, format='binary_folder', folder='analyzer_TDC_binary')
+
+
+
+.. parsed-literal::
+
+    estimate_sparsity:   0%|          | 0/10 [00:00<?, ?it/s]
 
 
-SpikeInterface provides a efficient way to extract waveforms from paired
-recording/sorting objects. The ``extract_waveforms`` function samples
-some spikes (by default ``max_spikes_per_unit=500``) for each unit,
-extracts, their waveforms, and stores them to disk. These waveforms are
+This folder is where all the postprocessing data will be saved such as
+waveforms and templates. Let’s calculate some waveforms. When doing
+this, the function samples some spikes (by default
+``max_spikes_per_unit=500``) for each unit, extracts their waveforms,
+and stores them to disk in ``extensions/waveforms``. These waveforms are
```

Comment:
Is the `analyzer_TDC_binary/extensions/waveforms` the full (relative) path here?

Reply:
Yes. The tree is `whatever-you-named-your-analyzer/extensions/waveforms/waveform-stuff`. For `spike_locations` it would be `whatever-you-named-your-analyzer/extensions/spike_locations/spike_locations-stuff`. Or did I misunderstand your question?

Reply:
Yeah, it was just confusing to me where `extensions/waveforms`.

So the full path would be. "path_from_where_i_run_the_file/how_i_named_the_analyzer/extensions/something"

I am not sure what would be more clear that just having the relative path with respect to the `analyzer_TDC_binary`. Maybe we can just write relative to `analyzer_TDC_binary` ?

Reply:
Yes, that makes sense to me. Maybe also adding the linux dot for current directory `./` as @zm711 suggests?

Reply:
Sounds good. I've changed to './analyzer_TDC_binary/extension/waveforms"